docs/pages/blog/introducing-pigment-css.md

```diff
@@ -0,0 +1,160 @@
+
+| Metrics                | Emotion | Pigment CSS | Reduction |
+| :--------------------- | ------: | ----------: | --------: |
+| First load JavaScript  |   131kB |       104kB |       20% |
+| First Contentful Paint |   503ms |       455ms |        9% |
+| Time To First Byte     | 447.5ms |     381.5ms |       15% |
+| Total Page HTML        |  15.9kB |      14.7kB |      7.5% |
+
+:::info
+Curious about where those performance numbers came from?
+[Check out this app](https://pigment-css-demo.vercel.app/perf) that compares Pigment CSS, Emotion, and styled-components across a variety of tests.
+:::
+
+### Familiar developer experience
+
+For developers migrating from Emotion or styled-components, you're probably already familiar with the most common patterns employed by Pigment CSS.
+`styled()` and `css()` are the two main functions used to define styles, and they mostly work the same as you'd expect them to (with some notable differences due to the nature of build-time CSS-in-JS—see [Coming from Emotion or styled-components](https://github.com/mui/pigment-css/tree/master?tab=readme-ov-file#coming-from-emotion-or-styled-components) for details).
+
+```jsx
+import { styled, css } from '@pigment-css/react';
+
+const Title = styled('h1') ({
+  fontSize: '2rem';
+});
+
+const Container = styled.div`
+  border: 1px solid red;
+
+  &:hover {
+    border-color: blue;
+  }
+
+  ${Title} {
+    margin-bottom: 2.5rem;
+  }
+`;
+
+export default function Modal() {
+  return (
+    <Container>
+        <Title>Hello</title>
+        <p className={css({ color: 'pink' })}>World</p>
+    </Container>
+  );
+}
+```
+
+We've also ported over [the `sx` prop](/system/getting-started/the-sx-prop/) from MUI System, so you can still define styles directly in a given component, but now it's much more performant than before.
+And in Pigment CSS, we've extended support for `sx` to include _all_ DOM nodes—not just Material UI components—so you don't need to wrap a simple `<div>` or `<span>` with a Box component to apply theme styles to it.
+
+```jsx
+<section sx={{ p: 2, border: '1px solid', borderColor: 'divider' }}>
+  <h1 sx={{ fontSize: '2rem', fontWeight: 700, mb: 1 }}>
+    Introducing Pigment CSS: the next generation of CSS-in-JS
+  </h1>
+  <p sx={{ color: 'text.primary', fontWeight: 500 }}>
+    Pigment CSS offers significant performance gains along with RSC
+  </p>
+</section>
+```
+
+### Future-proof solution
+
+Though we're still quite early in the RSC era, it seems inevitable that the React ecosystem as a whole will converge on this new paradigm over time.
+Next.js gave us our first glimpse with the App Router; RedwoodJS [recently released their own implementation](https://redwoodjs.com/blog/rsc-now-in-redwoodjs); and many other frameworks and meta-frameworks (like Remix) are currently working out [POCs and RFCs](https://github.com/remix-run/remix/discussions/8048) to catch up.
```

Comment:
Are we compatible with it?

Reply:
We are compatible with latest release of Remix which is a Vite plugin.